docs/docs/get-started/installing-nebari.md

```diff
@@ -45,6 +44,13 @@ There are currently two ways to install Nebari:
    mamba install nebari
    ```
 
+:::note
+The version of the Nebari your `nebari-config.yaml` must match your Nebari deployer package version.  See [Upgrading Nebari][nebari-upgrade] for techniques for upgrading your Nebari CLI or safely updating your older config file to match your Nebari package version.
+:::
+
+:::note
+The Nebari CLI will auto-detect and then deploy any [Nebari Extensions][nebari-extension-system] that are installed in your Python environment, and extensions once deployed cannot be uninstalled our automatically destroyed with a `nebari destroy` command.  For this reason, we recommend creating a [unique environment][environment-management] for each Nebari deployment, especially when managing multiple deployments with extensions.
```

Comment:
This note should've been moved to the extension docs, along with an FAQ on removing extensions and managing their installs. Also, while I don't have an example to corroborate this statement, a stage extension should be able to be destroyed by terraform, as they do have a `destroy` abstract method in the Stage class to handle it.

Reply:
The "no uninstall" part of the note is already mentioned in the extension docs, but I think it's a critical enough point to mention again here, since if you mistakenly deploy an extension (especially if it is in the default namespace), removing everything that gets installed is a headache.

Regarding destroying the extension - I've never seen an extension fully destroyed.  The pods and such are of course gone when EKS is blown away but things like S3 buckets and IAM roles linger.   A quick look at `destroy()` in `NebariTerraformStage` seems to me that it should work without the extension needing to override the method... so maybe there's a bug in the hook into extensions when destroy is called.  But I'll remove that part from the doc since at best it seems like it would be documenting a bug.  Next time I destroy, I'll watch the output more closely to see if the extension is called.